test/suite/wtthread.py

```diff
@@ -83,18 +83,16 @@ def run(self):
 
                 # TODO: We want a self.assertTrue here - be need to be a
                 # wttest to do that..
-                if not compare_tables(
-                        self, sess, uris, "checkpoint=WiredTigerCheckpoint"):
-                    print("Error: checkpoint tables differ.")
-                else:
-                    wttest.WiredTigerTestCase.printVerbose(
-                        3, "Checkpoint tables match")
 
-                if not compare_tables(self, bkp_session, uris):
-                    print("Error: backup tables differ.")
-                else:
-                    wttest.WiredTigerTestCase.printVerbose(
-                        3, "Backup tables match")
+                # Add an assert to stop running the test if any difference in table contents
+                # is found.
+                wttest.WiredTigerTestCase.printVerbose(3, "Testing if checkpoint tables match:")
+                assert compare_tables(self, sess, uris) == True
```

Comment:
You want to use `self.assertTrue()` for these two asserts.

Reply:
Thanks Sue! I'll fix this now.

Reply:
@sueloverso I just realised that there's a comment in the code saying "# TODO: We want a self.assertTrue here - be need to be a # wttest to do that..". Do we still want to transition this to self.assertTrue?

Reply:
First, please fix `be need to be` to a grammatically correct sentence and remove the double period at the end of the sentence. Second, I do not understand what the issue is and how `wttest` is involved. Doesn't `self.assertTrue` just evaluate the expression it is given? Is it because of the `printVerbose` calls? Those could be `self.pr` calls instead of verbose so that we always know where we are in the test. Anyway, can you give `self.assertTrue` a try and see if it works. If not, fix the comment to elaborate on why `self.assertTrue` does not work because using `assert` is uncommon.

Reply:
No worries, I'll fix that comment up. I don't think it is related to the printVerbose calls. Calling self.assertTrue results in an error: `AttributeError: 'backup_thread' object has no attribute 'assertTrue'`. That is probably why the code did not have any assert/self.assertTrue statements previously. I can remove the assert statements as well if that is preferred, let me know what you think.

Reply:
I think keeping the asserts are fine then because we want the thread to stop. Please add a detailed comment about why it cannot use the self.assertTrue though.

Reply:
Alright, comment has been added.